assets/scss/hero.scss

```diff
@@ -195,3 +195,9 @@
   }
 }
 
+.entity-page {
```

Comment:
Shouldn't this be in the entity page view itself, as it won't be used elsewhere?

Reply:
yes, thats correct, I've now moved it